source/meta.tex

```diff
@@ -1356,6 +1356,11 @@
 \begin{note}
 If a type has padding bits, the condition does not hold;
 otherwise, the condition holds true for integral types.
+If \tcode{std::numeric_limits<float>::is_iec559} is \keyword{true}, the
+condition does not hold because there are multiple object representations for
+some values, such as zero.
```

Comment:
This suggests that `has_unique_object_representations_v<T>` is `false` for all `T` when `std::numeric_limits<float>::is_iec559` is `true`. The three sentences of this note all use the phrase "the condition" with slightly different meanings.

Reply:
Good point. I've fixed this by consistently using `T` and "holds for `T`" in the updated wording.